pkg/kubernetes/client.go

```diff
@@ -198,6 +198,7 @@ func (c Client) GetService(svc service.MeshService) *corev1.Service {
 	return nil
 }
 
+//  TODO(whitneygriffith): remove or call provider.ListServices
```

Comment:
This function will actually stay, but be called from an encapsulating function provider.ListServices